cirq/vis/heatmap.py

```diff
@@ -28,24 +27,28 @@
 
 from cirq.devices import grid_qubit
 
-QubitCoordinate = Union[Tuple[int, int], grid_qubit.GridQubit]
+QubitCoordinate = Union[Tuple[float, float], grid_qubit.GridQubit]
```

Comment:
why did we change this to float, float? The qubits are themselves still on an integer grid!